src/main/java/mate/academy/spring/dto/MovieSessionRequestDto.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.dto;
+
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.Movie;
+
+public class MovieSessionRequestDto {
+    private Movie movie;
+    private CinemaHall cinemaHall;
+    private String showTime;
```

Comment:
```suggestion
    private Long movieId;
    private Long cinemaHallId;
    private String showTime;
```
If I'm not mistaken we must use only id

Reply:
agree

Reply:
done